src/robusta/runner/main.py

```diff
@@ -18,6 +19,11 @@ def main():
     loader = ConfigLoader(registry, event_handler)
     if os.environ.get("ENABLE_MANHOLE", "false").lower() == "true":
         manhole.install(locals=dict(getmembers(robusta_api)))
+
+    sentry_dsn = os.environ.get("SECRET_SENTRY_DSN", "")
+    if sentry_dsn:
```

Comment:
Sentry initialization might fail here in case of corrupted url.
I think we don't want to stop the runner initialization if that happens.

I think we need to catch that error, log it, and continue to work without sentry